retriever/lib/engine.py

```diff
@@ -227,7 +238,8 @@ def auto_get_datatypes(self, pk, source, columns, column_values):
         # set default column data types as int
         column_types = [('int',)] * len(columns)
         max_lengths = [0] * len(columns)
-
+        if column_values:
```

Comment:
This variable is redundant, I will consider revising the the parameters later to find out why I previously included it and never used it. For now I suggest keeping it here for a bit, if that is okey